scripts/files/templatesV3.1.ts

```diff
@@ -58,27 +58,27 @@ export const templates: {
   11155111: {
     eth: {
       bridge: '0x331Edb39Eb823602484cD9F28bBC34cE4B9e76D9',
-      sequencerInbox: '0x9a4332b77d150AB19388e7F01B686De91a3Eadf1',
+      sequencerInbox: '0x79C5c3c8355daDC0d48D37eDC3Cb5ac73cB5b9d2',
```

Comment:
The current `sequencerInbox` contract does not have the same code as on the branch. The correct address is in [this](https://github.com/EspressoSystems/nitro-contracts/commit/a4c6478e37b27d11e4bedfa8d2f3f836c957348a) commit.

Reply:
Hmm, I see it on etherscan? https://sepolia.etherscan.io/address/0x79C5c3c8355daDC0d48D37eDC3Cb5ac73cB5b9d2

Reply:
You can try searching for these

Reply:
Didn't you say your rollup was on ArbSepolia?

Reply:
My rollup was on arbsepolia but I have updated the addresses for both sepolia and arbsepolia. `11155111` signifies the chain id of sepolia